packages/calcite-components/src/components.d.ts

```diff
@@ -4715,6 +4715,7 @@ export namespace Components {
          */
         "messages": TableHeaderMessages;
         "numberCell": boolean;
+        "parentRowIsSelected": boolean;
```

Comment:
Testing it already, eh? ✨🦾🤖✨

Reply:
Unintentional QA === :+1: GREAT SUCCESS :+1: